judgels-backends/uriel/uriel-api/src/main/java/judgels/uriel/api/contest/web/ContestWebConfig.java

```diff
@@ -5,11 +5,13 @@
 import java.util.Optional;
 import java.util.Set;
 import judgels.uriel.api.contest.clarification.ContestClarificationStatus;
+import judgels.uriel.api.contest.role.ContestRole;
 import org.immutables.value.Value;
 
 @Value.Immutable
 @JsonDeserialize(as = ImmutableContestWebConfig.class)
 public interface ContestWebConfig {
+    ContestRole getContestRole();
```

Comment:
So the convention is, inside "contest" context, the field name does not need to have `contest` prefix anymore (but the type still needs it)